RaveBusinessLogic/BRAT.xml

```diff
@@ -9,28 +9,28 @@
         <Children collapsed="true">
           <Node xpathlabel="Name" xpath="SQLiteDB" type="file" />
           <Node xpathlabel="Name" xpath="HTMLFile" type="report" />
-          <Node label="Capacity" id="capacity">
+          <Node label="Capacity">
             <Children>
-              <Node label="Existing Dam Capacity" xpath="Vector" type="line" symbology="oCC_EX" />
+              <Node label="Existing Dam Capacity" xpath="Vector" type="line" symbology="oCC_EX" id="ex_capacity" />
               <Node label="Existing Dam Size" xpath="Vector" type="line" symbology="mCC_EX_CT" />
-              <Node label="Historic Dam Capacity" xpath="Vector" type="line" symbology="oCC_HPE" />
+              <Node label="Historic Dam Capacity" xpath="Vector" type="line" symbology="oCC_HPE" id="his_capacity" />
               <Node label="Historic Dam Size" xpath="Vector" type="line" symbology="mCC_HPE_CT" />

```

Comment:
These lines don't seem to do anything and are doubling the lines that come below